deployment/aws/cdk/config.py

```diff
@@ -85,4 +85,7 @@ class StackSettings(BaseSettings):
     # Default: - No specific limit - account limit.
     max_concurrent: Optional[int] = None
 
+    # The VPC Endpoint ID
+    vpc_endpoint_id: str = "{{VPCE_ID}}"
```

Comment:
This should be optional (default to None), we can then add a check within the `TitilerPrivateApiStack` construct

Reply:
Changed the config to optional and adapted the construct to it. Should be good.